### PR TITLE
Remove VPA Feature Gate

### PR DESCRIPTION
**Pull request by wyb1**
**What this PR does / why we need it**:
Removes the VPA feature gate. The vpa will now always be deployed.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```action operator
The `VPA` feature gate has been entirely removed. `VPA` components will now always be deployed. You have to remove the `VPA` feature gate from your gardener-controller-manager configuration after upgrading Gardener.
```

### CODE REVIEW

File: pkg/operation/seed/seed.go [hunk 1]
```diff
@@ -172,22 +172,19 @@ func generateWantedSecrets(seed *Seed, certificateAuthorities map[string]*utilse
 		)
 	}
 
-	// VPA feature gate
-	if controllermanagerfeatures.FeatureGate.Enabled(features.VPA) {
-		secretList = append(secretList,
-			&utilsecrets.CertificateSecretConfig{
-				Name: "vpa-tls-certs",
+	secretList = append(secretList,
```
Comment (rfranzke) [inline pkg/operation/seed/seed.go]: Can you add this directly to the above `secretList` instead of using `append()` here?

**Review by rfranzke — CHANGES_REQUESTED** [review]
Should we also remove https://github.com/gardener/gardener/blob/master/charts/seed-bootstrap/requirements.yaml#L15 ?

Should we also remove 
https://github.com/gardener/gardener/blob/master/pkg/operation/common/types.go#L165-L166 now?

**Comment by wyb1** [comment]
Not sure how I missed those. I removed them, thanks!

**Review by rfranzke — APPROVED** [review]
/lgtm

**Pull request merged**